### PR TITLE
Removes OpenShift Azure release v4 from CI

### PR DESCRIPTION
**Pull request by m1kola**
v4 is no longer supported, so removing all jobs related to that release.

### CODE REVIEW

File: projects/azure/RELEASES.md [hunk 1]
```diff
@@ -7,10 +7,11 @@ All Azure Red Hat OpenShift (ARO) versions:
...
 |release-v2|*         |deprecated |
 |release-v3|v3.2      |deprecated |
-|release-v4|v4.2      |live       |
-|release-v4|v4.3      |live       |
-|release-v4|v4.4      |pre-release|
-|release-v5|v5.1      |pre-release|
+|release-v4|v4.2      |deprecated |
+|release-v4|v4.3      |deprecated |
+|release-v4|v4.4      |deprecated |
+|release-v5|v5.1      |live       |
```
Comment (jim-minter) [inline projects/azure/RELEASES.md]: fwiw I think v5.1 is deprecated

**Comment by jim-minter** [comment]
one nit, also you can remove
```
ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-openshift-v4.yaml
ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-openshift-v4-presubmits.yaml
```

**Comment by jim-minter** [comment]
and grep `periodic-ci-azure-openshift-v4-create` - check with @mjudeikis about removing that too

**Comment by jim-minter** [comment]
ignore me - I'm wrong

File: projects/azure/RELEASES.md (hunk 1 above)
Comment (m1kola) [reply projects/azure/RELEASES.md]: Updated

**Comment by jim-minter** [comment]
"release/openshift" confusion on my part

/lgtm
/hold cancel

**Pull request merged**